cmake/modules/VTA.cmake

```diff
@@ -101,7 +101,9 @@ elseif(PYTHON)
        ${VTA_TARGET} STREQUAL "ultra96")
       target_link_libraries(vta ${__cma_lib})
     elseif(${VTA_TARGET} STREQUAL "de10nano")  # DE10-Nano rules
-      target_compile_definitions(vta PUBLIC VTA_MAX_XFER=2097152) # (1<<21)
+     #target_compile_definitions(vta PUBLIC VTA_MAX_XFER=2097152) # (1<<21)
```

Comment:
should we remove the line?

Reply:
I only left the comment there for reference, I suppose it can go.

Reply:
Great thanks!